Runtime/Scripts/UniformMaps/MetalRough2StandardMap.cs

```diff
@@ -5,15 +5,33 @@ namespace UnityGLTF
 	public class MetalRough2StandardMap : StandardMap, IMetalRoughUniformMap
 	{
 		private Vector2 baseColorOffset = new Vector2(0, 0);
+		private static readonly int TintId = Shader.PropertyToID("_Tint");
+		private static readonly int ColorId = Shader.PropertyToID("_Color");
+		private static readonly int BaseColorId = Shader.PropertyToID("_BaseColor");
+		private static readonly int BaseColorMapId = Shader.PropertyToID("_BaseColorMap");
```

Comment:
I just realized: 
I think the Shader properties _Tint and _BaseColorMap you keep adding everywhere are properties in our custom shaders, right?

If that is the case, i would prefer to change the names of the properties in our shaders to more standard names instead of bloating the plugin code with them